CONTRIBUTING.md

```diff
@@ -44,7 +44,7 @@ We are excited that you want to contribute to GUAC! This contributor ladder
 outlines different contributor roles within the project, along with
 responsibilities and privileges that come with them.
 
-Since GUAC is a complex project, there are 5 topic areas of interest:
+Since GUAC is a complex project, there are 5 software topic areas of interest:
```

Comment:
Require N-1 maintainer approval

Reply:
I think for these changes, it would be helpful to get some additional community feedback to see if this helps other users contribute better as well. I suggest that perhaps when this gets to a place where we think its ready, we can share it in the mailing lists / community meeting.

Reply:
That makes sense. I posted it in Slack when I opened the issue, but more attention on it wouldn't hurt. I'll put it on the agenda for the 20 June Community Meeting. If we put it off to the July meeting, we're probably bikeshedding too much.

I will say up front that I wouldn't be surprised if it doesn't move the needle much for the existing community members (although it might!). It's more of a foundation-laying step for trying to grow the community and bring in new contributors.

Reply:
I think june meeting is fine! agree, this shouldnt block till july